libcxx/test/std/containers/views/mdspan/extents/ctad.pass.cpp

```diff
@@ -21,6 +21,12 @@
 #include "../ConvertibleToIntegral.h"
 #include "test_macros.h"
 
+struct S {
```

Comment:
```
struct NonDefaultIndex {
   size_t value;
   constexpr NonDefaultIndex() = delete;
   constexpr NonDefaultIndex(size_t val) : value(val){}
   constexpr operator size_t() const noexcept { return value; }
 };
```

This will make it more obvious what we're testing.

Reply:
changed. I used `NoDefaultCtorIndex`